website/docs/docs/build/unit-tests.md

```diff
@@ -0,0 +1,253 @@
+
+Updating the regex logic to `'^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}$'` (those pesky escape characters) and rerunning the unit test solves the problem:
+
+```bash
+
+dbt test --select test_is_valid_email_address
+16:09:11  Running with dbt=1.8.0-a1
+16:09:12  Registered adapter: postgres=1.8.0-a1
+16:09:12  Found 6 models, 5 seeds, 4 data tests, 0 sources, 0 exposures, 0 metrics, 410 macros, 0 groups, 0 semantic models, 1 unit test
+16:09:12  
+16:09:13  Concurrency: 5 threads (target='postgres')
+16:09:13  
+16:09:13  1 of 1 START unit_test dim_wizards::test_is_valid_email_address ................... [RUN]
+16:09:13  1 of 1 PASS dim_wizards::test_is_valid_email_address .............................. [PASS in 0.26s]
+16:09:13  
+16:09:13  Finished running 1 unit_test in 0 hours 0 minutes and 0.75 seconds (0.75s).
+16:09:13  
+16:09:13  Completed successfully
+16:09:13  
+16:09:13  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
+
+```
+
+Your model is now ready for production! Adding this unit test helped catch an issue with the SQL logic _before_ you materialized `dim_customers` in your warehouse and will better ensure the reliability of this model in the future. 
+
+### Unit testing versioned models
+
+When a unit test is added to a model, it will run on all versions of the model by default.
+Using the example in this article, if you have versions 1, 2, and 3 of `my_model`, the `my test_is_valid_email_address` unit test will run on all 3 versions.
+
+To only unit test a specific version (or versions) of a model, include the desired version(s) in the model config:
+
+```yml
+
+unit-tests:
+  - name: test_is_valid_email_address # this is the unique name of the test
+    model: my_model # name of the model I'm unit testing
+      versions:
+        include: 
+          - 2
+    given: # optional: list of inputs to provide as fixtures
+
+```
+
+In this scenario, if you have version 1, 2, and 3 of `my_model`, `my test_is_valid_email_address` unit test will run on _only_ version 2.
+
+To unit test all versions except a specific version (or versions) of a model, you can exclude the relevant version(s) in the model config:
+
+```yml
+
+unit-tests:
+  - name: test_is_valid_email_address # this is the unique name of the test
+    model: my_model # name of the model I'm unit testing
+      versions:
+        exclude: 
+          - 1
+    given: # optional: list of inputs to provide as fixtures
+
+```
+So, if you have versions 1, 2, and 3 of `my_model`, your `test_is_valid_email_address` unit test will run on _only_ versions 2 and 3.
+
+If you want to unit test a model that references the pinned version of the model, you should specify that in the `ref` of your input:
+
+```yml
+
+unit-tests:
+  - name: test_is_valid_email_address # this is the unique name of the test
+    model: my_model # name of the model I am unit testing
+    given: # optional: list of inputs to provide as fixtures
+
+```
+
+### When to add a unit test to your model
```

Comment:
I think we kind of want to stress as much as possible that this is only available for model so that there is no confusion about language between these and data tests. It might benefit us to keep this language

Reply:
OK sounds good!